Makefile

```diff
@@ -50,13 +50,17 @@ nginx/cert.pem: nginx/key.pem
 secretkey:
 	openssl rand -hex 24
 
-build_dev:
+jetstream_config:
```

Comment:
Is this only ever going to be used for a jetstream git submodule? If not, it could be handy to name this just for purpose (e.g. `init_git_submodules`). Maybe someday we want to roll in other git submodules if only for further shared-configuration integration testing?

Reply:
Yeah we might, I liked having it explicitly declare that building depends on jetstream config, we could rename it if that comes up good point 👍